python/tvm/target/target.py

```diff
@@ -139,6 +142,14 @@ def current(allow_none=True):
         """
         return _ffi_api.TargetCurrent(allow_none)
 
+    def enter_cuda_scope(self):
+        if self.arch:
+            set_cuda_target_arch(self.arch)
+
+    @property
+    def arch(self):
```

Comment:
Again, you should not use target attributes in this way. You have to register this attribute to the CIDA target in the C++ side and use it here like others. Otherwise we cannot access this attribute in C++.

Reply:
There is alreadya  `arch` attribute in target class in the C++ side, and I am simply adding a python API of it.  
If it is not CUDA arch, `target.arch` will be None by default.